lib/search/helpArticleSearch.ts

```
@@ -0,0 +1,41 @@
+import Fuse from 'fuse.js';
```

ESLint

[error] Replace `'fuse.js'` with `"fuse.js"`

(prettier/prettier)